installation/resources/crds/istio/istios.operator.crd.yaml

```diff
@@ -0,0 +1,135 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: istios.operator.kyma-project.io
+spec:
+  group: operator.kyma-project.io
+  names:
+    kind: Istio
+    listKind: IstioList
+    plural: istios
+    singular: istio
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: Istio is the Schema for the istio API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: IstioSpec defines the desired state of Istio
+            properties:
+              config:
+                description: Configurion.
```

Comment:
Wrong description. Why don't you put the number of proxies directly in spec? What else do you plan to put at the same level as config? Any particular reason for that element?

Reply:
@videlov please improve description of this field